lib/simctl.js

```diff
@@ -736,7 +736,7 @@ async function getDeviceTypes () {
 /**
  * Get the full list of runtimes, devicetypes, devices and pairs in JSON
  *
- * @return {Array<string>} List of the device types, runtimes devices and pairs JSO
+ * @return {Object} Object containing device types, runtimes devices and pairs JSON
```

Comment:
Not sure that this is technically correct. It is not returning JSON, but an Object. I'm not sure JSON needs to be mentioned here at all.